src/moh_kenya_echis/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="app_name">eCHIS Kenya</string>
+	<bool name="allowCustomHosts">False</bool>
```

Comment:
should we add this property to every project? or if the fallback value when building a branded app is always `False`, should we remove it here?

Reply:
So, `allowCustomHosts` only is relevant if you _do not_ have an `app_host` value.  For normal brands that only target a single CHT server, you should put the URL for the servers as the `app_host` value. When `app_host` exists, then the server-list page (for choosing which server to connect to) is never shown.  

To be able to use your branded app (or the Unbranded app) to connect to _multiple different CHT servers_, you should not set the `app_host` value.  That will cause the Server-list page to be displayed when the app is run for the first time.  The `allowCustomHosts` flag just controls whether or not the server-list will allow users to enter a custom URL (instead of just picking one from this list).

We want the _Unbranded_ app to allow custom hosts. So, in the `main` config we set `allowCustomHosts = true` (since that is where the unbranded app gets all its config).  Any other brand that has an `app_host` value (so all of them besides `moh_kenya_echis`) does not need to set a value.

Reply:
It makes a lot of sense, thanks a lot for the detailed explanation @jkuester!